pray4ramadan-porch.php

```diff
@@ -343,6 +323,35 @@ function dt_hook_ajax_notice_handler(){
     }
 } );
 
+if ( ! function_exists( 'p4r_get_theme_hex' ) ) {
```

Comment:
this function misses the custom_theme_color.
instead use the `PORCH_COLOR_SCHEME_HEX` constant that is calculate once.